cpp/src/gandiva/projector.h

```diff
@@ -138,6 +168,8 @@ class GANDIVA_EXPORT Projector {
   SchemaPtr schema_;
   FieldVector output_fields_;
   std::shared_ptr<Configuration> configuration_;
+  bool compiled_from_cache_;
+  static size_t used_cache_size_;
```

Comment:
This is static so all the projector will use the same var to keep track of all projectors' memory consumption. But now, I'm thinking to remove it and only keep tracking of the global used memory by the cache, instead of both tracking.

Reply:
Removed it as it will no longer be used.